httpapi/server.go

```diff
@@ -0,0 +1,26 @@
+package httpapi
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/julienschmidt/httprouter"
```

Comment:
Choosing routers is like choosing your favorite color for a car...but have you looked at the `chi` router before? It's quite versatile...

https://github.com/go-chi/chi

Reply:
I used `httprouter` because it's used in EG. No other reason and IMHO we should care about that right now.